graphbolt/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ if(USE_CUDA)
     ${BOLT_DIR}/cuda/*.cu
   )
   list(APPEND BOLT_SRC ${BOLT_CUDA_SRC})
+  if(DEFINED ENV{CUDAARCHS})
+    set(CMAKE_CUDA_ARCHITECTURES $ENV{CUDAARCHS})
```

Comment:
Do we need to set it explicitly? According to the extracted from https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_ARCHITECTURES.html: Initialized by the [CUDAARCHS](https://cmake.org/cmake/help/latest/envvar/CUDAARCHS.html#envvar:CUDAARCHS) environment variable if set.

Reply:
Only for cmake version greater than 3.20. to remove this line, we need to increase minimum cmake version to 3.20.

Reply:
I added it because the current minimum is 3.18.

Reply:
Also, for some reason, the `find_package(Torch REQUIRED)` line empties the contents of the `CMAKE_CUDA_ARCHITECTURES` variable, so I had to place this after that line.

Reply:
Issue tracking this: https://github.com/pytorch/pytorch/issues/115937